README.md

```diff
@@ -1,2 +1,5 @@
-# Project-review
-I have to review a project I completed during Microverse admission trials
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Good work pointing this out :+1: A descriptive `README` helps users to have a better understanding of what the project is about. However, it would be better if you refrained from giving orders and made suggestions instead to sound nicer and friendlier :smile: